src/Compilers/CSharp/Portable/Parser/LexerCache.cs

```diff
@@ -181,23 +183,32 @@ internal bool TryGetKeywordKind(string key, out SyntaxKind kind)
             return kind != SyntaxKind.None;
         }
 
-        internal SyntaxTrivia LookupTrivia<TArg>(
-            char[] textBuffer,
-            int keyStart,
-            int keyLength,
-            int hashCode,
-            Func<TArg, SyntaxTrivia> createTriviaFunction,
-            TArg data)
+        internal SyntaxTrivia LookupWhitespaceTrivia(
+            SlidingTextWindow textWindow,
+            int lexemeStartPosition,
+            int hashCode)
```

Comment:
easier to just read as the new function.  There was only ever one caller ofr LookupTrivia, and it always used the same generic arg.